### PR TITLE
Psionic Content

### PR DESCRIPTION
**Pull request by ZeroDayDaemon**
## About the PR 
Adds a few pieces of psionic clothing
:cl: Daemon
- add: Added new psionic clothing.

### CODE REVIEW

File: Content.Server/Nyanotrasen/Abilities/Psionics/Abilities/NoosphericZapPowerSystem.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        private void OnShutdown(EntityUid uid, NoosphericZapPowerComponent component, ComponentShutdown args)
+        {
+            if (_prototypeManager.TryIndex<EntityTargetActionPrototype>("NoosphericZap", out var pacify))
+                _actions.RemoveAction(uid, new EntityTargetAction(pacify), null);
+        }
+
+        private void OnPowerUsed(NoosphericZapPowerActionEvent args)
+        {
+            if (!(TryComp<DamageableComponent>(args.Target, out var damageable) && damageable.DamageContainerID == "Biological"))
```
Comment (Elijahrane) [inline Content.Server/Nyanotrasen/Abilities/Psionics/Abilities/NoosphericZapPowerSystem.cs]: your test component should be if they have PotentialPsionicComponent

File: Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+- type: entity
+  parent: ClothingHeadsetAlt
+  id: ClothingHeadsetBinaural
+  name: binaural overear-headset
+  description: a headset with a binaural audio feed, careful, you might hear voices in your head. Placeholder Sprite.
+  components:
+  - type: Headset
+    channels:
+    - Telepathic
```
Comment (Elijahrane) [inline Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml]: telepathic isn't a channel

**Review by Elijahrane — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  description: a headset with a binaural audio feed, careful, you might hear voices in your head. Placeholder Sprite.
+  components:
+  - type: Headset
+    channels:
+    - Common
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/syndicate.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/syndicate.rsi
+#Functionality not coded yet
```
Comment (Elijahrane) [inline Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml]: dont really want placeholders for these

Comment (Elijahrane) [reply Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml]: just delete the headsets

**Comment by ZeroDayDaemon** [comment]
Yeah tests are failing, yeah its the tests fault not my code

**Comment by Elijahrane** [comment]
the failures look related to your code

**Comment by ZeroDayDaemon** [comment]
> the failures look related to your code

Well you're lookin wrong then

**Comment by Elijahrane** [comment]
get rid of the copy + pasted pacify variable names
  and Trycomp out var damageable in the zap power

**Comment by ZeroDayDaemon** [comment]
> get rid of the copy + pasted pacify variable names and Trycomp out var damageable in the zap power

You also need to fix your variable names for powers, since I didnt even base this on pacify

**Comment by Elijahrane** [comment]
already did

**Review by Elijahrane — APPROVED** [review]

**Pull request merged**